### PR TITLE
Integrate NIBittensorLLM 

### PR DESCRIPTION
**Pull request by BitVoyagerMan**
- Add NIBittensorLLM endpoint based on power of Bittensor.

### CODE REVIEW

File: langchain/src/llms/bittensor.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+  /** Call out to NIBittensorLLM's complete endpoint.
+   Args:
+       prompt: The prompt to pass into the model.
+
+       Returns:
+   The string generated by the model.
+
+   Example:
+   let response = ai21._call("Tell me a joke.");
```
Comment (jacoblee93) [inline langchain/src/llms/bittensor.ts:56]: Update string to bittensor

File: langchain/package.json [hunk 2]
```diff
@@ -637,7 +640,7 @@
     "@xata.io/client": "^0.25.1",
     "@zilliz/milvus2-sdk-node": ">=2.2.11",
     "apify-client": "^2.7.1",
-    "axios": "^0.26.0",
+    "axios": "^0.26.1",
```
Comment (jacoblee93) [inline langchain/package.json:643]: Can we do this with just `fetch`? Otherwise people will need to install packages to use this

**Comment by jacoblee93** [comment]
Thanks for this! It would be better to try to do this without optional deps - it looks like we can probably do this with fetch?

File: langchain/src/llms/bittensor.ts (hunk 1 above)
Comment (BitVoyagerMan) [reply langchain/src/llms/bittensor.ts:56]: Oh, thanks. will fix that comment.

File: langchain/package.json (hunk 2 above)
Comment (BitVoyagerMan) [reply langchain/package.json:643]: Which is better between using axios@0.26.0 or fetch?
Because axios@0.26.0 is also working for me.

Comment (BitVoyagerMan) [reply langchain/package.json]: I resolved with fetch

**Pull request closed without merging**